spt/strafestuff.cpp

```diff
@@ -1028,7 +1042,7 @@ namespace Strafe
 	{
 		if (StrafeJump(jumped, player, vars, ducking, out))
 		{
-			if (!yawChanged || tas_strafe_allow_jump_override.GetBool())
+			if ((!yawChanged || tas_strafe_allow_jump_override.GetBool()) && !out.Processed)
```

Comment:
Is this change related?

Reply:
the other jump commands run the MapSpeeds function that sets the fmove and smove for that frame so that needs to be skipped with this method

Reply:
Would you prefer if this got refactored into the StrafeJump function instead?

Reply:
I see. Idk, whatever you think is better.

Reply:
I think I'll refactor it